app/services/price_policy_updater.rb

```diff
@@ -39,10 +44,53 @@ def assign_attributes
 
   def save
     ActiveRecord::Base.transaction do
-      @price_policies.all?(&:save) || raise(ActiveRecord::Rollback)
+      (assign_attributes && @price_policies.all?(&:save)) || raise(ActiveRecord::Rollback)
```

Comment:
@LeticiaErrandonea should this change be dependent on the product pricing mode?

Reply:
Actually, I think this method is only called on line 33, so we should revert this change